### DISCUSSION TITLE
postprocessor/csv output of CZM interface materials

### DISCUSSION THREAD

**Discussion by mcacace** [idaholab/moose#28337] [Q&A Modules: Solid mechanics]
Dear all,
I'm doing some work with the CZM interface implementation, but could not find a proper way to output material property values that are defined on the boundary interface into a csv. Or better, I can export them from the exodus file (extracting along a line from Paraview), but I'd prefer to output those directly via a (vector)postprocessor (or similar) at specified locations (to maintain flexible output intervals for the full model solution and the postprocessor). Does such a functionality exist in the framework? Thanks for any hint/clue etc ... 
Mauro

**Comment by GiudGiud** [comment]
Hello

What you want is the equivalent of this for a boundary
https://mooseframework.inl.gov/source/vectorpostprocessors/MaterialVectorPostprocessor.html

We unfortunately do not have this implemented. It would be very useful and we would love to have that.
Would you want to develop this?

Guillaume
